### PR TITLE
Appbidding googleadmanager criteo sdk 4.0 migration

### PR DESCRIPTION
**Pull request by gbakatsias**
Updated googleadmanager app bidding sample app migrating to Criteo SDK 4.0 with live bidding

### CODE REVIEW

File: appbidding_googleadmanager/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -2,7 +2,9 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="com.criteo.publisher.samples.appbidding_googleadmanager">
 
-  <application
+    <uses-permission android:name="android.permission.INTERNET" />
```
Comment (irwansetiawan) [inline appbidding_googleadmanager/src/main/AndroidManifest.xml]: Why do we need to explicitly add `INTERNET` permission here?

Criteo SDK [permissions](https://github.com/criteo/android-publisher-sdk/blob/3af93645383fc8079bd9615e5b40e9d5e08fda07/publisher-sdk/src/main/AndroidManifest.xml#L5) have been defined in the SDK's manifest and will be merged into the app's manifest on build.

File: appbidding_googleadmanager/build.gradle [hunk 2]
```diff
@@ -21,5 +21,5 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.1.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
     implementation 'com.google.android.gms:play-services-ads:19.1.0'
-    implementation 'com.criteo.publisher:criteo-publisher-sdk:3.10.0'
+    implementation 'com.criteo.publisher:criteo-publisher-sdk:4.0.0'
```
Comment (irwansetiawan) [inline appbidding_googleadmanager/build.gradle:24]: Please also update `versionName` (a few lines above) to `4.0.0.0`

**Review by irwansetiawan — CHANGES_REQUESTED** [review]

File: appbidding_googleadmanager/src/main/AndroidManifest.xml (hunk 1 above)
Comment (gbakatsias) [reply appbidding_googleadmanager/src/main/AndroidManifest.xml]: Indeed not necessary. Removed it

File: appbidding_googleadmanager/build.gradle (hunk 2 above)
Comment (gbakatsias) [reply appbidding_googleadmanager/build.gradle:24]: Updated

**Review by Amokrane — APPROVED** [review]

**Review by irwansetiawan — APPROVED** [review]

**Pull request merged**